src/Symfony/Bundle/TwigBundle/Resources/views/Exception/exception_full.html.twig

```diff
@@ -1,7 +1,136 @@
 {% extends '@Twig/layout.html.twig' %}
 
 {% block head %}
-    <link href="{{ absolute_url(asset('bundles/framework/css/exception.css')) }}" rel="stylesheet" type="text/css" media="all" />
+    <style>
+        .sf-reset .traces {
+            padding-bottom: 14px;
```

Comment:
I agree, this is indeed what is done elsewhere, but here, I don't want to "pollute" the namespace as there are only exceptions templates, and creating a new directory up sounds like not really useful.